mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,64 @@
+import {useState} from "react";
+import clsx from "clsx";
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ * {@link Checkbox}
+ */
+interface CheckboxProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default checkbox status
+   * @default false
+   */
+  isDefaultChecked?: boolean;
```

Comment:
Does it responsible for only default status, or also for current? Looks like after changing it state of checkbox will change too

Reply:
We need this property if we want to display a checkbox on a page with the status already set to checked, because we encapsulate the state inside the component and we may need both a checked checkbox and an empty one during the initial render.

Reply:
But better change name to isChecked

Reply:
rewrote